src/components/ListItem.js

```diff
@@ -53,6 +56,9 @@ const ListItem = ({ item, token }) => {
       >
         Purchase
       </button>
+      <button onClick={handleClick}>
```

Comment:
This call to `handleClick` is causing the build to fail—it's called without having been defined.

Reply:
It's a work in progress. We have this solved. We are trying to solve our error message `TypeError: event.preventDefault is not a function`

We had an infinite loop when we clicked our trash can icon and used the `event.preventDefault` to stop that but now we get this error message. We are a little stuck right now.